File: BE/WebParent/BE_Client/src/main/java/com/pbl5/client/dto/StoreRequestDto.java
package com.pbl5.client.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class StoreRequestDto {
    private String storeName;
    private String address;
    private String phoneNumber;
    private String description;
    private Integer customerId;

    public StoreRequestDto(Integer id, String storeName, String address, String description, String phoneNumber, Integer id1, int status, Date requestDate) {
    }
}

File: FE/FE_Client/src/models/dto/StoreRequestDto.ts
export interface StoreRequestDto {
    customerId: number;
    storeName: string;
    description: string;
    phoneNumber: string;
    address: string;
}

export interface StoreRequestDtoImpl extends StoreRequestDto {
    status: number;
    responseDate: Date;
    requestDate: Date;
    responseNote: string;
}